branching_xblock/static/js/src/branching_xblock.js

```diff
@@ -64,6 +94,15 @@ function BranchingXBlock(runtime, element) {
       }).done(updateView);
     }
 
+    $el.on('toggle', '[data-role="hint-collapsible"]', function() {
+        const open = this.open;
+        isHintVisible = Boolean(open);
+        const $summary = $(this).find('[data-role="hint-summary"]');
+        if ($summary.length) {
+            $summary.text(open ? 'Hide hint' : 'Show hint');
+        }
+    });
+
```

Comment:
This logic seems to be a partial repeat of the `setHintVisibility`. Maybe the toggle event listener can be mapped to the `setHintVisibility`?

Reply:
Done.